Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -141,6 +148,14 @@ NetworkStorageManager::NetworkStorageManager(PAL::SessionID sessionID, IPC::Conn
 {
     ASSERT(RunLoop::isMain());
 
+    if (!path.isEmpty()) {
+        auto existingManager = activePaths().get(path);
+        if (!existingManager)
+            activePaths().set(path, this);
+        else
+            RELEASE_LOG_ERROR(Storage, "%p - NetworkStorageManager::NetworkStorageManager path for session %" PRIu64 " is already in use by session %" PRIu64, this, m_sessionID.toUInt64(), existingManager->sessionID().toUInt64());
```

Comment:
You can probably write it as:
```
auto addResult = activePaths().add(path, *this);
RELEASE_LOG_ERROR_IF(! addResult.isNewEntry, ...)
```

We could probably only do this work if RELEASE_LOG is defined, ditto for activePaths().

And maybe add a FIXME: remove once rdar.... is fixed